pkg/iac/scanners/terraform/ignore_test.go

```diff
@@ -56,7 +56,7 @@ resource "bad" "my-rule" {
 }
 `, assertLength: 0},
 		{name: "IgnoreLineAboveTheBlockMatchingParamBool", inputOptions: `
-// tfsec:ignore:*[secure=false]
+// trivy:ignore:*[secure=false]
```

Comment:
I would still keep the old tfsec tests in so that we remain backwards compatible but add another test case for the trivy ignores.

Reply:
I seem to have renamed it by accident, fixed it https://github.com/aquasecurity/trivy/pull/6358/commits/dd269431c74cff4183a87c930f31e05e8860f771